### ISSUE TITLE
Missing URL-Encoding of ids or anchor elements

### ISSUE THREAD

**Issue by N7K4** [toshimaru/jekyll-toc#95]
I have in my TOC the German word "Datenschutzerklärung" (engl. Privacy) with an ä inside.

When I create the TOC the href or the anchor are getting `#datenschutzerkl%C3%A4rung`

```
<a class="anchor" href="#datenschutzerkl%C3%A4rung" aria-hidden="true"><span class="octicon octicon-link"></span></a>
```

In the url bar of my browser I see https://example.com/#datenschutzerklärung.

To harmonize the results and be secure for other special chars, I would suggest to use url_encode for the ID-Element and the Anchor-Link.

**Comment by mpg-toshimaru-enomoto** [comment]
> To harmonize the results and be secure for other special chars, I would suggest to use url_encode

Anchor-link works without url encode.

So, I'm wordering what is the security concern❓ Do we really have to encode special chars❓

**Comment by toshimaru** [comment]
> To harmonize the results and be secure for other special chars, I would suggest to use url_encode

Anchor-link works without url encode.

So, I'm wordering what is the security concern❓ Do we really have to encode special chars❓

**Comment by N7K4** [comment]
There is no security concern. I just want that the URL can be safely used. With URL encoding the links in HTML and address bar are the same

**Comment by toshimaru** [comment]
I believe this is fixed v0.14.0. Thanks for reporting issue!
